packages/react/src/patterns/blocks/CalloutQuote/__tests__/CalloutQuote.e2e.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Defines the host for testing
+ *
+ * @type {string | string}
+ * @private
+ */
+const _url =
+  process?.env.SELENIUM_HOST || 'https://ibmdotcom-react-canary.mybluemix.net';
+
+/**
+ * Sets the correct path
+ *
+ * @type {string}
+ * @private
+ */
+const _path = '/iframe.html?id=patterns-blocks-calloutquote--default';
+
+describe('CalloutQuote', () => {
+  beforeAll(() => {
+    browser.url(_url + _path);
+    browser.setWindowSize(1200, 800);
+    browser.pause(1000);
+  });
```

Comment:
@jeffchew @asudoh Regarding testing UI, perhaps something like this? 

```suggestion
  beforeAll(() => {
    document.addEventListener("DOMContentLoaded", function(event) {
      browser.url(_url + _path);
      browser.setWindowSize(1200, 800);
      browser.pause(1000);
      done();
    });
  });
```

Reply:
@kennylam Exactly, the notion is tracking for events that lets us proceed to the next step. It'll depend on each wait calls seen in this PR. And agreed with @jeffchew that we can do it in separate PRs.